tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/ErrorHelpers.cs

```diff
@@ -112,4 +115,31 @@ private static string SanitizeTagValue(string tag)
         SpanTagHelper.TryNormalizeTagName(tag, normalizeSpaces: true, out var normalizedTag);
         return normalizedTag;
     }
+
+    private static async Task SkipKnownCrashes(string pathToCrashReport, ITestOutputHelper output)
+    {
+        if (pathToCrashReport == null || !File.Exists(pathToCrashReport))
+        {
+            return;
+        }
+
+        try
+        {
+            using var crashReport = new CrashReport(pathToCrashReport, output);
+            var stacktrace = await crashReport.ResolveCrashStackTrace();
+
+            output.WriteLine("Crash stacktrace:");
+
+            foreach (var frame in stacktrace)
```

Comment:
It might be better to use a for loop here. because we are already looking through the stacktrace inside `ResolveCrashStackTrace()` it seems

https://github.com/DataDog/dd-trace-dotnet/blob/68b277a40d8782d444bc8a72852336c0f9eff8e0/tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/CrashReport.cs#L38-L48

Reply:
Sorry I don't understand your point